.github/CODEOWNERS

```
@@ -3,379 +3,596 @@
+Modules/_sqlite/              @berkerpeksag @erlend-aasland
 
-# Config Parser
-Lib/configparser.py           @jaraco
-Lib/test/test_configparser.py @jaraco
+# subprocess
+Lib/subprocess.py             @gpshead
+Lib/test/test_subprocess.py   @gpshead
+Modules/*subprocess*          @gpshead
+
+# Tarfile
+Doc/library/tarfile.rst       @ethanfurman
+Lib/tarfile.py                @ethanfurman
+Lib/test/test_tarfile.py      @ethanfurman
+
+# Tomllib
+Doc/library/tomllib.rst       @encukou @hauntsaninja
+Lib/test/test_tomllib/        @encukou @hauntsaninja
+Lib/tomllib/                  @encukou @hauntsaninja
+
+# Typing
+Doc/library/typing.rst        @JelleZijlstra @AlexWaygood
+Lib/test/test_typing.py       @JelleZijlstra @AlexWaygood
+Lib/test/typinganndata/       @JelleZijlstra @AlexWaygood
+Lib/typing.py                 @JelleZijlstra @AlexWaygood
+Modules/_typingmodule.c       @JelleZijlstra @AlexWaygood
+
+# Types
+Lib/test/test_types.py        @AA-Turner
+Lib/types.py                  @AA-Turner
+Modules/_typesmodule.c        @AA-Turner
 
-# Doc sections
-Doc/reference/                @willingc @AA-Turner
-Doc/whatsnew/                 @AA-Turner
+# Unittest
+Lib/unittest/mock.py          @cjw296
+Lib/test/test_unittest/testmock/  @cjw296
 
-**/*weakref*                  @kumaraditya303
+# Urllib
+**/*robotparser*              @berkerpeksag
 
-# Colorize
-Lib/_colorize.py              @hugovk
-Lib/test/test__colorize.py    @hugovk
+# Venv
+**/*venv*                     @vsajip @FFY00
 
-# Fuzzing
-Modules/_xxtestfuzz/          @ammaraskar
+# Weakref
+**/*weakref*                  @kumaraditya303
 
-# t-strings
-**/*interpolationobject*      @lysnikolaou
-**/*templateobject*           @lysnikolaou
-**/*templatelib*              @lysnikolaou @AA-Turner
-**/*tstring*                  @lysnikolaou
+# Zipfile.Path
+Lib/test/test_zipfile/_path/  @jaraco
+Lib/zipfile/_path/            @jaraco
 
-# Remote debugging
-Python/remote_debug.h         @pablogsal
-Python/remote_debugging.c     @pablogsal
-Modules/_remote_debugging_module.c @pablogsal @ambv @1st1
+# Zstandard
+Lib/compression/zstd/         @AA-Turner
+Lib/test/test_zstd.py         @AA-Turner
+Modules/_zstd/                @AA-Turner
 
-# gettext
-**/*gettext*                  @tomasr8
+# ----------------------------------------------------------------------------
 
-# Internal Docs
-InternalDocs/                 @AA-Turner
+# Exclusions from .github/CODEOWNERS should go at the very end
+# because the final matching pattern will take precedence.
+
+# Exclude .mailmap from being owned by @python/email-team
+.mailmap
+
+# Exclude Argument Clinic directories
+Modules/_ctypes/clinic/
+Modules/_io/clinic/
+Modules/_multiprocessing/clinic/
+Modules/_sqlite/clinic/
+Modules/_sre/clinic/
+Modules/_ssl/clinic/
+Modules/_testcapi/clinic/
+Modules/_testinternalcapi/clinic/
+Modules/_testlimitedcapi/clinic/
+Modules/_zstd/clinic/
+Modules/cjkcodecs/clinic/
```

Can these be grouped like this?
```suggestion
Modules/**/clinic/
```

---

Good idea, done